react/components/molecules/item/item.js

```diff
@@ -63,34 +65,39 @@ export class Item extends PureComponent {
                 <Touchable
                     style={this._contentStyle()}
                     activeOpacity={this.props.activeOpacity}
+                    underlayColor={this.props.underlayColor}
                     disabled={this.props.disabled}
                     onPress={this.props.onPress}
                 >
-                    {this.props.header && (
-                        <LinearGradient
-                            style={styles.gradient}
-                            angle={this.props.gradientAngle}
-                            colors={this.props.gradientColors}
-                            locations={this.props.gradientLocations}
-                            useAngle={true}
-                        >
-                            <View style={styles.header}>
-                                <View style={styles.headerTextContainer}>
-                                    <Icon
-                                        icon={this.props.headerIcon}
-                                        width={17}
-                                        height={17}
-                                        color="#ffffff"
-                                    />
-                                    <Text style={styles.headerText}>{this.props.headerText}</Text>
+                    <>

```

Comment:
why this specific change?

Reply:
This was a bug fix since `Item` should not pass several childs to component. It was throwing if I set an underlayColor color

Reply:
Got it, nice!